modules/social_features/social_user/social_user.install

```diff
@@ -38,6 +38,14 @@ function social_user_update_8001(&$sandbox) {
 }
 
 /**
+ * Add view, block users permissions for the content,site managers.
+ */
+function social_user_update_8002() {
+  _social_user_set_permissions();
```

Comment:
The referenced function will nuke existing permissions for contentmanagers and can cause problem in enterprise sites or other existing sites build on Open Social.

Reply:
Did you check this locally? Seems it will only change(makes them as from box) permissions, that related to this module:
    'access user profiles',
    'cancel account',
    'change own username',
    'select account cancellation method',
and
   'view users',
    'block users',

You could check code for function `_social_user_set_permissions` to make more complex conclussion.